website/docs/developer/development-guide/code/integration-tests.md

```diff
@@ -122,30 +131,41 @@ This is a list of tests that should be created for existing Orca architecture. T
 - [Error](#error-paths):
   1. Requests for catalog info on a non-existent granule/file should return HTTP Status 404.
 
+  1. Attempting to ingest a file that does not exist should result in `files does not exist`.
+
 ### Recovery
 - [Happy](#happy-path):
   1. Ingest randomized data to Orca.
      :::tip
-     Include a 200 GB file to make sure timeouts do not prevent recovery.
+     Include a large (bigger than 250 Gb) file to make sure timeouts do not prevent recovery.
      :::
   1. Call the OrcaRecoveryWorkflow to restore the files from Orca to another bucket.
+     :::tip
+     Make sure to cover ExcludeFileTypes being set, being unset, and excluding files. May require additional tests.
+     :::
   1. Retry calls to the [Recovery Granules API](../../../developer/api/api-gateway.md/#recovery-granules-api) until entries are found, and status is `complete`.
      :::warning
      Recovery may take up to 4 hours.
      :::
-  1. Verify that the files are present in the proper Orca bucket.
+  1. Verify that the files are present in the proper target bucket.
 - [Security](#security-paths):
+  1. Follow the Happy test up to contacting the API.
+  1. No recovery endpoints should be publicly accessible, even if the granule-id or other values are known.
+
+  :::tip
+  The following test will only be valid after completing [ORCA-351](https://bugs.earthdata.nasa.gov/browse/ORCA-351).
+  :::
   1. Follow the Happy test up to calling the workflow.
   1. Workflow should not be publicly accessible, even if files are valid.
   1. Follow the Happy test up to calling the workflow.
-  1. Workflow should not be able to restore files to arbitrary buckets.
-  1. Follow the Happy test up to contacting the API.
-  1. No recovery endpoints should be publicly accessible, even if the granule-id or other values are known.
+  1. Workflow should not be able to restore files to arbitrary buckets that are not registered with Orca.
```

Comment:
I don't feel this is correct yet. So we pass where it is and where we want it, and depending of we are talking about current behavior or future behavior, the resulting bucket we copy to can be arbitrary. Thoughts?

Reply:
My understanding is that https://github.com/nasa/cumulus-orca/blob/develop/modules/iam/main.tf locks us to buckets that are registered with Orca.
If there is going to be a discussion about whether we keep that implementation, or move to being able to restore to literally any bucket, then we should wipe this theoretical test.